src/docs/sdk/data-handling.mdx

```diff
@@ -1,23 +1,44 @@
 ---
 title: "Data Handling"
-sidebar_order: 5
+sidebar_order: 3
 ---
 
 Data handling is the standardized context in how we want SDKs help users filter data.
 
 ## Sensitive Data
 
-In older SDKs you might sometimes see elaborate constructs to allow the user to strip away sensitive data. Newer SDKs no longer have this feature as it turned out to be too hard to maintain per-SDK. Instead, only two simple config options are left:
+SDKs should not include PII or other sensitive data in the payload by default.
 
-- [_send-default-pii_](https://docs.sentry.io/error-reporting/configuration/#send-default-pii) is **disabled by default**, meaning that data that is naturally sensitive is not sent by default. That means, for example:
+- [_send-default-pii_](https://docs.sentry.io/platforms/python/configuration/options/#send-default-pii) is **disabled by default**, meaning that data that is naturally sensitive is not sent by default. That means, for example:
 
   - When attaching HTTP requests to events, "raw" bodies (bodies which cannot be parsed as JSON or formdata) are removed, and known sensitive headers such as `Authorization` or `Cookies` are removed too.
 
   - User-specific information (e.g. the current user ID according to the used webframework) is not sent at all.
 
-  - Note that if a user explicitly sets a request on the scope, nothing is stripped from that request. The above rules only apply to integrations that come with the SDK.
+  - On desktop applications, the username logged in the device is not included.
 
-- [_before-send_](https://docs.sentry.io/error-reporting/configuration/#before-send) can be used to register a callback with custom logic to remove sensitive data.
+  - SDKs don't send `{{auto}}` as `user.ip`. This instructs the server to keep the connection's IP address.
```

Comment:
May be worth clarifying that client SDKs will always have an IP captured as they directly connect to the Sentry service to submit data. And also if we note this, just put an analogy that it is similar to how server-side logs work, and the data is captured for auditibility.

Reply:
Great point about the server-side logging, I'll add.
Wrt Sentry already capturing, AFAIK this is only true for iOS and JavaScript events (based on `event.platform=javascript` or `cocoa`. Every other platform requires `user.ip={{auto}}`